src/commands/synthetics/run-test.ts

```diff
@@ -19,27 +20,52 @@ export class RunTestCommand extends Command {
     pollingTimeout: 2 * 60 * 1000,
     proxy: {protocol: 'http'} as ProxyConfiguration,
     subdomain: process.env.DATADOG_SUBDOMAIN || 'app',
+    tunnel: false,
   }
   private configPath?: string
   private publicIds: string[] = []
+  private shouldOpenTunnel?: boolean
   private testSearchQuery?: string
 
   public async execute() {
     const startTime = Date.now()
+    const stdoutLogger = this.context.stdout.write.bind(this.context.stdout)
 
     this.config = await parseConfigFile(this.config, this.configPath)
 
     const api = this.getApiHelper()
-    const publicIdsTriggers = this.publicIds.map((id) => ({config: this.config.global, id}))
-    const testsToTrigger = publicIdsTriggers.length ? publicIdsTriggers : await this.getTestsToTrigger(api)
+    const publicIdsFromCli = this.publicIds.map((id) => ({config: this.config.global, id}))
+    const testsToTrigger = publicIdsFromCli.length ? publicIdsFromCli : await this.getTestsToTrigger(api)
+    const publicIdsToTrigger = testsToTrigger.map(({id}) => id)
 
     if (!testsToTrigger.length) {
       this.context.stdout.write('No test suites to run.\n')
 
       return 0
     }
 
-    const {tests, triggers} = await runTests(api, testsToTrigger, this.context.stdout.write.bind(this.context.stdout))
+    let tunnel: Tunnel | undefined
+    if (this.config.tunnel || this.shouldOpenTunnel) {
```

Comment:
Don't we want one of these setting superseding the other?

Like if I have the tunnel setting in my config, but I decide to overwrite it with the flag for instance, it would always work as long as one of the two is truthy.

Reply:
Agreed - I updated it to have the CLI flag superseeding the configuration.